pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/bloom/BloomIndexType.java

```diff
@@ -50,6 +50,7 @@
 public class BloomIndexType
     extends AbstractIndexType<BloomFilterConfig, BloomFilterReader, BloomFilterCreator>
     implements ConfigurableFromIndexLoadingConfig<BloomFilterConfig> {
+  public static final String INDEX_DISPLAY_NAME = "bloom";
```

Comment:
Is @VisibleForTesting better here? Same for the other types of indexes?

Reply:
It is a good practice to make these constants public. Specially in this case, where they can be useful if a code external to this class wants to analyze the TableConfig